rest-api-spec/build.gradle

```diff
@@ -82,6 +82,7 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("search.aggregation/20_terms/string profiler via map", "The profiler results aren't backwards compatible.")
   task.skipTest("search.aggregation/20_terms/numeric profiler", "The profiler results aren't backwards compatible.")
   task.skipTest("migration/10_get_feature_upgrade_status/Get feature upgrade status", "Awaits backport")
+  task.skipTest("cluster.health/20_request_timeout/cluster health request timeout with 200 response code", "Backwards incompatible, returns a warning in 8.0")
```

Comment:
I think we shouldn't be adding to the list of skipped tasks. Instead, we can use the `allowed_warnings` feature to say that this warning is allowed, but not required, in the compatibility tests.

Reply:
Created https://github.com/elastic/elasticsearch/pull/80178 to backport the change to 7.16 and then I can enable the test again, so and the BWC tests won't be broken.